.eslintrc

```diff
@@ -8,6 +8,7 @@
   },
   "env": {
     "node": true,
+    "es6": true
```

Comment:
I guess I can remove that. I added it so es6 would stop fussing about Promises, but since I've removed Promises from the code, it's not necessary anymore.